docker/terraform/README.md

```diff
@@ -0,0 +1,27 @@
+# Terraform Cloud Builder Image
+Used for running Terraform in Cloud Build. Contains the required `terraform`, `gcloud`, and `jq` executables.
```

Comment:
What is `jq` used for?

Reply:
It's used by a the [`esp_full_version`](https://github.com/michaelkedar/osv.dev/blob/terraform-cloudbuild/deployment/terraform/modules/osv/scripts/esp_full_version) script to interface with [Terraform inputs/outputs](https://github.com/michaelkedar/osv.dev/blob/6ead7f770340b70144c69e82af85c1acd0115dbc/deployment/terraform/modules/osv/osv_api.tf#L55-L58) (which are in json)

Actually, the output is simple enough that I can probably avoid using it. I might remove it.